checker.cpp

```diff
@@ -1,46 +1,13 @@
-#include <iostream>
 #include <assert.h>
-#include <unistd.h>
-using namespace std;
-
-int vitalsOk(float temperature, float pulseRate, float spo2) {
-  if(temperature > 102 || temperature < 95) {
-    cout << "Temperature critical!\n";
-    for (int i = 0; i < 6; i++)
-    {
-      cout << "\r* " << flush;
-      sleep(1);
-      cout << "\r *" << flush;
-      sleep(1);
-    }
-
-    return 0;
-  } else if(pulseRate < 60 || pulseRate > 100) {
-    cout << "Pulse Rate is out of range!\n";
-    for (int i = 0; i < 6; i++)
-    {
-      cout << "\r* " << flush;
-      sleep(1);
-      cout << "\r *" << flush;
-      sleep(1);
-    }
-    return 0;
-  } else if(spo2 < 90) {
-    cout << "Oxygen Saturation out of range!\n";
-    for (int i = 0; i < 6; i++)
-    {
-      cout << "\r* " << flush;
-      sleep(1);
-      cout << "\r *" << flush;
-      sleep(1);
-    }
-    return 0;
-  }
-  return 1;
-}
+#include "HealthChecker.h"
+#include "Language.h";
+                                
+std::string language = "German";
 
 int main() {
-  assert(!vitalsOk(99, 102, 70));
-  assert(vitalsOk(98.1, 70, 98));
-  cout << "Done\n";
+    languageFileName(language);
```

Comment:
The function name `languageFileName` sounds like it converts the language into the filename.
It does that, as well as loads the file.
You could have two lines - getting the filename and loading it. And maybe in future, another for handling the error when the file doesn't exist